C++/syntax_test_cpp.cpp

```diff
@@ -2753,3 +2753,11 @@ void sayHi()
 /**
       *
 /*    ^ comment.block.c punctuation.definition.comment.c */
+
+typedef struct MyStructure {} MyStructure_t;
```

Comment:
The same here. There's a test case at line 410, which this one could moved to for better grouping. The current one at 410 could also be tweaked to test your change.